newrelic/hooks/mlmodel_openai.py

```diff
@@ -780,15 +789,15 @@ def _record_events_on_stop_iteration(self, transaction):
         self._nr_ft.__exit__(None, None, None)
         try:
             openai_attrs = getattr(self, "_nr_openai_attrs", {})
-
             # If there are no openai attrs exit early as there's no data to record.
             if not openai_attrs:
                 return
 
             completion_id = str(uuid.uuid4())
+            llm_context_attrs = getattr(self, "nr_llm_context_attrs", None)
```

Comment:
Given that we missed this one-we are probably missing a test for this case too:
```suggestion
            llm_context_attrs = getattr(self, "_nr_llm_context_attrs", None)
```